### PR TITLE
Feat/156 Select Application Screen part two

### PR DESCRIPTION
**Pull request by basilv**
Progress on #156, will likely need a 3rd PR after this one though.

### CODE REVIEW

**Comment by basilv** [comment]
More refinements coming...

File: frontend/public/env.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+window.localStorage.setItem('fam_api_base_url', 'https://341ihp76l2.execute-api.ca-central-1.amazonaws.com/test')
```
Comment (ianliuwk1019) [inline frontend/public/env.js:1]: Just quick question.
I understand at the moment this is hardcoded.
Will later the value come from Terraform?

File: frontend/src/App.vue [hunk 2]
```diff
@@ -1,19 +1,27 @@
 <script setup lang="ts">
+import { inject } from 'vue'
 import { RouterLink, RouterView } from 'vue-router'
+
+const environmentDisplayName = inject('fam_environment_display_name') as string
+var environmentLabel = ''
```
Comment (ianliuwk1019) [inline frontend/src/App.vue:6]: Does compiler complaint with using 'let' not 'var'?
Is 'inject' also get value from 'localstorage'?
Also is it possible for Vue to use 'environmentLable' directly for 'inject' (I mean so we don't need to declare two variables)?

**Review by ianliuwk1019 — APPROVED** [review]
Approved. Just few simple questions.

File: frontend/public/env.js (hunk 1 above)
Comment (basilv) [reply frontend/public/env.js:1]: Yes, it will come from the pipeline.

**Pull request merged**